pkg/ccl/streamingccl/settings.go

```diff
@@ -39,7 +40,20 @@ var StreamReplicationJobLivenessTimeout = settings.RegisterDurationSetting(
 	"stream_replication.job_liveness_timeout",
 	"controls how long we wait for to kill an inactive producer job",
 	3*24*time.Hour,
-	settings.WithName("stream_replication.job_liveness.timeout"),
+	settings.WithRetiredName("stream_replication.job_liveness.timeout"),
```

Comment:
you don't need to register a retired name if it's the same as the key. maybe worth clarifying as a comment on the WithRetiredName API function.

Reply:
It's not the same as the key though: liveness_timeout vs liveness.timeout.

Reply:
nice. carry on